src/main/java/remi/model/TaskList.java

```diff
@@ -0,0 +1,120 @@
+package remi.model;
+
+import remi.utils.RemiError;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Stores the list of Tasks, allows you to add, remove, and find tasks by index.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
```

Comment:
The recommended naming for collection is by using plural noun. But I am not sure this is bad 🤔.